app/components/admin/stats/box_component.rb

```diff
@@ -0,0 +1,9 @@
+class Admin::Stats::BoxComponent < ApplicationComponent
```

Comment:
I'm not sure what the name `Box` means in this context :thinking:. How about `StatComponent`?